hypershift-operator/controllers/hostedclustersizing/hostedclustersizing_controller_test.go

```diff
@@ -106,9 +106,9 @@ func TestSizingController_Reconcile(t *testing.T) {
 			},
 			nodePoolsForHostedCluster: func(_ context.Context, _ *hypershiftv1beta1.HostedCluster) (*hypershiftv1beta1.NodePoolList, error) {
 				return &hypershiftv1beta1.NodePoolList{Items: []hypershiftv1beta1.NodePool{
-					{Status: hypershiftv1beta1.NodePoolStatus{Replicas: 10}},
```

Comment:
I would expect to see a new unit test with autoscaling off, using spec, and this existing test would continue to use status and not have autoscaling on.